server/.babelrc

```diff
@@ -1,27 +1,32 @@
 {
   "presets": [
-    ["env", {
-      "targets": {
-        "browsers": [
-          "last 2 versions",
-          "ios_saf >= 8",
-          "ie >= 10",
-          "chrome >= 49",
-          "firefox >= 49",
-          "> 1%"
-        ]
-      },
-      "debug": false,
-      "loose": true,
-      "useBuiltIns": true
-    }],
-    "react"
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "browsers": [
+            "last 2 versions",
+            "ios_saf >= 8",
+            "ie >= 10",
+            "chrome >= 49",
+            "firefox >= 49",
+            "> 1%"
+          ]
+        },
+        "debug": false,
+        "loose": true,
+        "useBuiltIns": "entry"
```

Comment:
This used to be `"useBuiltIns": true`, but `true` is no longer a valid option for `@babel/preset-env` (see the [docs](https://babeljs.io/docs/en/babel-preset-env#usebuiltins)).

I chose `"entry"` here, which looks to inline parts of the `@babel/polyfill` that are required by the "browsers" option above (if I understand correctly).

Let me know if you have a preferred option here that isn't `"entry"` (again, the [docs](https://babeljs.io/docs/en/babel-preset-env#usebuiltins) for reference).